### PR TITLE
make Matrix(x) error messages correct and consistent

### PR DESCRIPTION
**Pull request by aardgoose**
pedant mode on.

### CODE REVIEW

File: src/math/Matrix3.js [hunk 1]
```diff
@@ -184,7 +184,7 @@ Object.assign( Matrix3.prototype, {
 
 		if ( matrix && matrix.isMatrix4 ) {
 
-			console.error( "THREE.Matrix3.getInverse no longer takes a Matrix4 argument." );
+			console.error( "THREE.Matrix3: .getInverse no longer takes a Matrix4 argument." );
```
Comment (Mugen87) [inline src/math/Matrix3.js]: It should be `.getInverse()` 😉

File: src/math/Matrix3.js [hunk 2]
```diff
@@ -203,7 +203,7 @@ Object.assign( Matrix3.prototype, {
 
 		if ( det === 0 ) {
 
-			var msg = "THREE.Matrix3.getInverse(): can't invert matrix, determinant is 0";
+			var msg = "THREE.Matrix3: .getInverse(): can't invert matrix, determinant is 0";
```
Comment (Mugen87) [inline src/math/Matrix3.js]: You can remove the colon right after `.getInverse()`

**Review by Mugen87 — CHANGES_REQUESTED** [review]

File: src/math/Matrix4.js [hunk 3]
```diff
@@ -555,7 +555,7 @@ Object.assign( Matrix4.prototype, {
 
 		if ( det === 0 ) {
 
-			var msg = "THREE.Matrix4.getInverse(): can't invert matrix, determinant is 0";
+			var msg = "THREE.Matrix4: .getInverse(): can't invert matrix, determinant is 0";
```
Comment (Mugen87) [inline src/math/Matrix4.js]: And here too...

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**